index.js

```diff
@@ -5,17 +5,7 @@ var config = require('./config');
 var _ = require('lodash');
 var hookUrl;
 
-var baseSlackMessage = {
-  channel: config.slackChannel,
```

Comment:
Are these variables removed completely? Or filled somewhere else?

Reply:
This variables are completely removed. I and @lnikell think that filling variables in Slack UI during a webhook configuration is a better way to customize a bot.

Reply:
Ah, great find! I didn't realize that. I think it might be a good idea, in the future, to consider attachments for better display of some information. I suppose, we really need better parsers in general to get the info that's needed out of generic messages.

Just so it's noted: my biggest goal with formatting the messages, was to **avoid generic message title** text, like "Lambda CloudWatch Notification". So, if you notice incoming messages that lack actionable formatting, don't hesitate to pull again! :)